apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,98 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  /**
+   * Demo saveCheckerResult function
+   *
+   * This function is called:
+   * - every time the conclusion page is loaded (without user data)
+   * - every time the user wants to save the progress (including user data)
+   * - every time the user wants to save the conclusion (including user data)
+   *
+   * This function is called from another file. This is just a sketch.
+   */
+  const saveCheckerResult = () => {
+
+    const mutationData = {
+
+      environmentUrl: window.location.href,
+      appVersion: process.env.REACT_APP_VERSION,
+      topic: topic,
+
+
+      // How in the world are we going to get STTR version(s)?
+      // In topic.sttrFile we know the JSON file, but that's it
+      // If we really want the STTR version, we can do that 2 ways:
+      // 1) We go into topics.source.json and go get all activity id's and with those go into the xml files
+      // 2) (Re)decide to add them into the [topic].json file
+      activities: [
+        {
+          id: 'monument activity',
+          version: 1.2.4,
+        },
+        {
+          id: 'building activity',
+          version: 1.3.2,
+        }
+      ],
+
+
+      // Get `questionAnswers()` from checker.js in branch feature/autofill PR#127
+      // We need our own unique question id's (questionHash?) and not the changing id's from FL
+      // `getQuestionAnswers()` example array:
+      // [
+      //   'chappie-unique-hash-839472340': "Yes, because",
+      //   'chappie-unique-hash-839472341': false,
+      //   'chappie-unique-hash-839472342': "No, because",
+      //   'chappie-unique-hash-839472343': true,
+      // ]
+      questionAnswers: getQuestionAnswers(),
+      // Question: Do we also want to store question text? I think that's overload.
+
+
+      // `getPermitConclusions()` example array:
+      // [{
+      //   title: "Conclusion title 1",
+      //   description: "Conclusion description 1",
+      // }]
+      permitConclusions: getPermitConclusions()
+
+
+      // See Trello card "Refactor Context"
+      registerLookups: {
+        monument: { },
+        cityScape: { },
+
+        bag: {
+          postalCode: '',
+          streetname: '',
+          // IMO, we should store all data, except for `houseNumberFull`
+          // Only store `houseNumberFull` when user agrees to save personal data
+          // All other data are free of personal information
+          houseNumberFull: '',
+          // ...
+        },
+      },
+
+      userInput: {
+
+        address: {
+          postalCode: postalCode,
+          streetName: streetName,
+          // IMO, we should store all data, except for `houseNumberFull`
+          // Only store `houseNumberFull` when user agrees to save personal data
+          // All other data are free of personal information
+          houseNumberFull: houseNumberFull,
```

Comment:
maybe split house number + addition because that's what the user actually inputs.